README.md

```diff
@@ -53,6 +53,9 @@ These steps are to be completed by the author of the blog post.
 5. Push the file to GitHub, then create a pull request (PR) into the `draft` branch.
 
    Link the PR to the issue you created in Step 1.
+   Anyone can review the PR but @lauracowen (or other admin) need to merge it.
```

Comment:
Is this statement true?  So far, it seemed that anyone can review the PR, and then then originator can merge it to `draft`.

Reply:
oh if that's the case (I can't tell), then that's perfect (and makes more sense). I'll just modify that text. Thanks.